test/multi_inspiral/bank_veto_bank.xml

```diff
@@ -0,0 +1,176 @@
+<?xml version='1.0' encoding='utf-8'?>
+<!DOCTYPE LIGO_LW SYSTEM "http://ldas-sw.ligo.caltech.edu/doc/ligolwAPI/html/ligolw_dtd.txt">
+<LIGO_LW>
+	<Table Name="processgroup:process:table">
+		<Column Type="lstring" Name="program"/>
+		<Column Type="lstring" Name="version"/>
+		<Column Type="lstring" Name="cvs_repository"/>
+		<Column Type="int_4s" Name="cvs_entry_time"/>
+		<Column Type="lstring" Name="comment"/>
+		<Column Type="int_4s" Name="is_online"/>
+		<Column Type="lstring" Name="node"/>
+		<Column Type="lstring" Name="username"/>
+		<Column Type="int_4s" Name="unix_procid"/>
+		<Column Type="int_4s" Name="start_time"/>
+		<Column Type="int_4s" Name="end_time"/>
+		<Column Type="int_4s" Name="jobid"/>
+		<Column Type="lstring" Name="domain"/>
+		<Column Type="lstring" Name="ifos"/>
+		<Column Type="int_8s" Name="process_id"/>
+		<Stream Name="processgroup:process:table" Type="Local" Delimiter=",">
+			"tmpltbank","645a41a2fa3df783221bd9988f70b0cee1861d56","UNCLEAN: some modifications were not commited!",946819435," ",0,"titan1.atlas.aei.uni-hannover.de","spxiwh",16799,956049146,956049411,0,"lalapps","H1",0
```

Comment:
The comment "UNCLEAN [...]" can be removed; it's from cvs days and would not look good on master

Reply:
This caught my eye! This file dates from:
```
(pycbc_test) [iwharry@harrymac relbin]$ lalapps_tconvert 946819435
Wed Jan 06 13:23:40 UTC 2010
```

Might be time for a new bank_veto bank ;)

Reply:
Thanks, @pannarale I've removed that now.

@spxiwh Yes, a new veto bank for O4 is on the to-do list!